src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
```

Comment:
Function getSum does not take two arguments as per the task description. It should take two arguments, not one.